### PR TITLE
[5.4] Support for calling artisan in different queues

### PR DESCRIPTION
**Pull request by treycheng**
```php
// Support for calling artisan in different queues. For example
Artisan::queue('test', ['id' => $id], "queue-2");
Artisan::queue('test', ['id' => $id], "queue-1");

```

### CODE REVIEW

File: src/Illuminate/Foundation/Console/Kernel.php [hunk 1]
```diff
@@ -222,14 +222,18 @@ public function call($command, array $parameters = [], $outputBuffer = null)
     /**
      * Queue the given console command.
      *
-     * @param  string  $command
-     * @param  array   $parameters
+     * @param  string $command
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/Console/Kernel.php]: Please don't change this alignment.

File: src/Illuminate/Foundation/Console/Kernel.php [hunk 2]
```diff
@@ -222,14 +222,18 @@ public function call($command, array $parameters = [], $outputBuffer = null)
...
+     * @param  array  $parameters
+     * @param  string $queue
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/Console/Kernel.php]: string|null

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

**Comment by yekexuan** [comment]
@GrahamCampbell Thank you. I changed the style.

**Comment by taylorotwell** [comment]
What if you want to change the connection too? Needs to be thought out more in that regards.

**Pull request closed without merging**